//公会踢人
message DceGuildKickMember
{
	optional string uid		= 1;//被踢uid
}